tasks/task963_librispeech_asr_next_word_prediction.json

```diff
@@ -0,0 +1,937 @@
+{
+    "Source": [
+        "librispeech_asr"
+    ],
+    "Definition": "Predict the next word given an incomplete English Sentence.",
+  
+    "Positive Examples": [ 
+        { 
+        "input": "How is it",
+        "output": "going?", 
+        "explanation": "Given the partial English sentence 'How is it', the user has generated 'going?' as an output. This is an excellent example because 'How is it going?' is a common phrase in English Langauge."
+        }, 
+        {   
+        "input": "United states is a",
+        "output": "country.",  
+        "explanation": "This is a good example because the user broke up an English fact."
+        }
+    ], 
+  
+    "Negative Examples": [ 
+        { 
+        "input": "Once up on a",
+        "output": "Lorry",  
+        "explanation": "This is a bad example because we need to predict 'time' as the next word to the given sentence, but we have predicted 'Lorry'"
+        }, 
+        {   
+        "input": "How the contry has",
+        "output": "23", 
+        "explanation": "This is a bad example because we need to predict 'changed' as the next word to the given sentence, but we have predicted '23'"
+        }
+    ],
+    "Instances": [
+        {
+            "input": "chapter one missus rachel lynde is surprised missus rachel lynde lived just where the avonlea",
+            "output": [
+                "main"
+            ]
+        },
+        {
+            "input": "that had its source away back in the woods of the old cuthbert place it was reputed to be an intricate headlong brook in its earlier course through those",
+            "output": [
+                "woods"
+            ]
+        },
+        {
+            "input": "for not even a brook could run past",
+            "output": [
+                "missus"
+            ]
+        },
+        {
+            "input": "and that if she noticed anything odd or out of place she would never rest until she had ferreted out the whys and wherefores thereof there are plenty of people in avonlea and out of it who can attend closely",
+            "output": [
+                "to"
+            ]
+        },
+        {
+            "input": "but missus rachel",
+            "output": [
+                "lynde"
```

Comment:
hmm... why is this a well-defined instance?

Reply:
I've made some further comments on this task, see below.